src/api/storage-volumes.tsx

```diff
@@ -218,6 +219,28 @@ export const deleteStorageVolume = async (
   ).then(handleResponse);
 };
 
+export const deleteStorageVolumeBulk = async (
+  volumes: LxdStorageVolume[],
+  project: string,
+): Promise<PromiseSettledResult<void>[]> => {
+  const results: PromiseSettledResult<void>[] = [];
+  return new Promise((resolve, reject) => {
+    Promise.allSettled(
+      volumes.map(async (volume) => {
+        return deleteStorageVolume(volume.name, volume.pool, project)
```

Comment:
```suggestion
        return deleteStorageVolume(volume.name, volume.pool, project, volume.location)
```
For a cluster we need to specify the target when deleting a volume as well to uniquely identify it.